### DISCUSSION TITLE
Which API to toggle the suggestion?

### DISCUSSION THREAD

**Discussion by trinhanhtuan** [mrcjkb/rustaceanvim#524] [Q&A]
Hi Al,
![Screenshot 2024-09-26 164747](https://github.com/user-attachments/assets/8df14acc-4a9e-4cbd-a6a7-86f2d7edd3a5)

I am using NvChad starter configuration with `rustaceanvim` added, things work fine. However I don't know which LSP function can be used to map with a hotkey to turn on/off the suggestions (the line with bubble in attached scree-shot) while all other checkings still work. Please help. Thank you.

**Comment by mrcjkb** [comment]
`:h vim.diagnostic`
